ddlite.py

```diff
@@ -78,6 +90,8 @@ class candidate_internal(object):
   See entity_internal and relation_internal for examples
   """
   def __init__(self, all_idxs, labels, sent, xt):
+    self.uid = "{}::{}::{}::{}".format(sent.doc_name, sent.sent_id,
```

Comment:
I see. thanks for the clarification! The doc_id field we have has a different semantics. I will change accordingly.